data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,426 @@ mission "Remnant: Face to Maw 2B"
+				accept
+			label remnantuntrusted
+			`	Aeolus smiles mysteriously at you. "Let's just say that you are not the only resourceful person working for us. We will share more with you once you have demonstrated your loyalty."`
+				accept
+	on complete
+		dialog
+			`It takes a few minutes to contact the yardmaster, but once you do, a mixed crew of Hai and Human dockworkers have the crates moved promptly from the warehouse to your cargo bay. Hai and Humans seem accustomed to working together here.`
+
+
+
+mission "Remnant: Crystal Research 2"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
+	landing
+	source "Greenwater"
+	destination "Allhome"
+	cargo "Keystones" 2
+	to offer
+		has "Remnant: Crystal Research 1: done"
+	on complete
+		payment 250000
+
+
+
+mission "Remnant: Crystal Research 3"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from several Hai worlds, ending on Newhome.`
+	landing
+	source "Allhome"
+	destination "Newhome"
+	stopover "Makerplace"
+	stopover "Stonebreak"
+	stopover "Giverstone"
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Icelake"
+	stopover "Heartvalley"
+	cargo "Keystones" 4
+	to offer
+		has "Remnant: Crystal Research 2: done"
+	on offer
+		conversation
+			`True to Aeolus' word, there are crates of Hai Quantum Keystones waiting for you. As you ensure they are securely loaded, your comm system informs you that a message has arrived. It has no indication of provenance, but appears to be a notification that a payment of 250,000 marked 'on-assignment allowance' has been made to your account.`
+			`	The attached message is brief and to the point. "Greetings, Captain <first>. Hopefully the shipments were waiting for you. We have made arrangements for samples to be acquired from Makerplace, Stonebreak, Giverstone, Cloudfire, Snowfeather, Icelake, and Heartvalley. Once you have collected all those, head to Newhome. There are a few other places that we are still working to set up, and we should have those ready by the time you reach Newhome."`
+				accept
+	on stopover
+		dialog
+			`On each stop you quickly locate and load the shipment of crystals. The packaging comes with brochures and images depicting comfortable-looking Hai relaxing on a spaceship observation deck. "Guaranteed to provide the smoothest flight for your passengers!" reads one of the labels.`
+	on complete
+		payment 250000
+		dialog
+			`You are just checking in with the port authorities to locate your shipment when your commlink indicates a pair of messages incoming. While you wait for the second one to load, you see the first message is a deposit statement indicating that a payment of <payment> has been deposited from a "Solar Winds Logistics Ltd."`
+
+
+
+mission "Remnant: Crystal Research 3A"
+	invisible
+	landing
+	source "Makerplace"
+	destination "Newhome"
+	cargo "Keystones" 2
```

Comment:
Thanks for the heads-up.

If I reserve all the space at the start of the mission, wouldn't that mean that by the end of it the player would have twice the space taken up (half by the "reserved" and half by the actual keystones?

Reply:
OK. As suggested, I've got all the missions set up so that it will appear on the first planet, and the rest will happen in sequence. I'm fine with the possibility of a slight desync.